contribs/gmf/src/authentication/Service.js

```diff
@@ -16,6 +16,7 @@ import olEventsEventTarget from 'ol/events/Target.js';
 
 /**
  * @typedef {Object} User
+ * @property {string|null} email User's email address
```

Comment:
```suggestion
 * @property {string} [email] User's email address
```
same for other properties

Reply:
That's not entirely true.  If you look at the code, you'll notice that it gets set to `null`.  `[email]` means `string|undefined`, not `string|null`.  It has not effect if one use a thruthy comparision, like `if (user.email)`, but would not work if `if (user.email === null)`.

For the sake of not breaking anything I'd leave the others as-is and this one like this too.

Reply:
OK, I will have a look :-)